src/components/Job/JobTicket.tsx

```diff
@@ -1,28 +1,66 @@
-import React from 'react'
-import './jobticket.scss'
-const JobTicket = () => {
+import React from "react";
+import "./jobticket.scss";
+import { JobModal } from "../../type/type";
+
+type Props = {
+  item: JobModal;
+};
+
+const JobTicket = ({ item }: Props) => {
+  const date = new Date(item.createdAt);
+  const newDate = new Date();
+
+  const dateFormat = (date: Date) => {
+    const currentDate = Math.abs((newDate.getDate() - date.getDate()) * 24);
```

Comment:
Please check name variable.
Exp:
date should be jobCreationDate
newDate -> currentDate
currentDate -> distanceBetweenToDate

Reply:
i've changed